README.md

```diff
@@ -303,35 +189,29 @@ while `f(?` is definitely a placeholder).
 # Grammar
 
 ```grammarkdown
-ArgumentList[Yield, Await]:
-  `?`
-  AssignmentExpression[+In, ?Yield, ?Await]
-  `...` AssignmentExpression[+In, ?Yield, ?Await]?
-  ArgumentList[?Yield, ?Await] `,` `?`
-  ArgumentList[?Yield, ?Await] `,` AssignmentExpression[+In, ?Yield, ?Await]
-  ArgumentList[?Yield, ?Await] `,` `...` AssignmentExpression[+In, ?Yield, ?Await]?
-```
+MemberExpression[Yield, Await] :
+  `new` MemberExpression[?Yield, ?Await] Arguments[?Yield, ?Await, +New]
+
+CallExpression[Yield, Await] :
+  CallExpression[?Yield, ?Await] Arguments[?Yield, ?Await, ~New]
 
-<!--
-# Out of Scope/Future Directions
-
-There are several additional features that are currently out of scope for this proposal, but may be 
-considered in future proposals or added to this proposal if there is a valid reason to do so:
-
-* Positional placeholders:
-  ```js
-  f(?1, ?0)         // roughly: (x, y) => f(y, x)
-  ```
-* Spread a placeholder *without* rest:
-  ```js
-  f(1, ...?, 2)     // roughly: (x) => f(1, ...x, 2)
-  ```
-* Default initializers (similar to initializers/defaults for parameters, destructuring, and 
-  binding patterns):
-  ```js
-  f(? = 1, 2)       // roughly: (x = 1) => f(x, 2)
-  ```
--->
+CoverCallExpressionAndAsyncArrowHead[Yield, Await]:
+  MemberExpression[?Yield, ?Await] Arguments[?Yield, ?Await, ~New]
+
+CallMemberExpression[Yield, Await] :
+  MemberExpression[?Yield, ?Await] Arguments[?Yield, ?Await, ~New]
+
+SuperCall[Yield, Await] :
+  `super` Arguments[?Yield, ?Await, ~New]
+
+Arguments[Yield, Await, New] :
```

Comment:
Editorial: Consider calling parameter something like `AllowPlaceholder` rather than `New` (as the permitted distribution of `?` is changing over various versions of this proposal).